src/Apps/Sell/Components/SubmissionLayout.tsx

```diff
@@ -31,13 +31,13 @@ export const SubmissionLayout: React.FC<SubmissionLayoutProps> = ({
             </Column>
 
             <Column span={[8]}>
-              <Box maxWidth="100vw" width={CONTENT_WIDTH} p={2} pt={4}>
+              <Box maxWidth={CONTENT_WIDTH} p={2} pt={[2, 4]}>
                 {children}
               </Box>
             </Column>
           </GridColumns>
         ) : (
-          <Box maxWidth="100vw" width={CONTENT_WIDTH} p={2} pt={4}>
+          <Box maxWidth={CONTENT_WIDTH} p={2} pt={[2, 4]}>
```

Comment:
`maxWidth={CONTENT_WIDTH}` is enough for responsiveness.

Reply:
This change is causing layout issues on the title step. I opened https://github.com/artsy/force/pull/14081 to revert it. Was there a specific reason for this change?

 
<img width="392" alt="Screenshot 2024-06-25 at 21 28 12" src="https://github.com/artsy/force/assets/4691889/2870c012-8421-4e91-bef0-7d87042dbda0">